MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,593 @@
+    <syncfusion:DataGridTemplateColumn.CellTemplate>
+        <DataTemplate>
+            <StackLayout  Margin="10">
+                <DatePicker Date="{Binding ShippedDate}" TextColor="Black"/>
+            </StackLayout>
+        </DataTemplate>
+    </syncfusion:DataGridTemplateColumn.CellTemplate>
+</syncfusion:DataGridTemplateColumn>
+{% endhighlight %}
+
+## DataGridDateColumn
+
+The DataGridDateColumn inherits all the properties of the `DataGridColumn`. It displays the date information as the content of a column. To create the `SfDataGrid.DataGridDateColumn`, the property corresponding to the column in the underlying collection must be of type DateTime. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage.BindingContext>
+    <local:ViewModel x:Name ="viewModel"/>
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="DataGrid" ItemsSource="{Binding OrderInfoCollection}" AutoGenerateColumnsMode="None">
+        <syncfusion:SfDataGrid.Columns >
+           <syncfusion:DataGridDateColumn Format="d"  HeaderText="Date" MappingName="ShippedDate" />
+        </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs"%}
+dataGrid = new SfDataGrid();
+DataGridDateColumn dateColumn = new DataGridDateColumn()
+{
+    MappingName = "ShippedDate",
+    HeaderText = "Date",
+    Format = "d"
+};
+dataGrid.Columns.Add(dateColumn);
+{% endhighlight %}
+{% endtabs %}
+
+## DataGridNumericColumn
+
+The `DataGridNumericColumn` inherits all the properties of `DataGridColumn`. It is used to display numeric data. To create `DataGridNumericColumn`, the property corresponding to the column in the underlying collection must be a numeric type(int, double, float, etc.). To create a `GridNumericColumn`, follow the code example:
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml"%}
+<ContentPage.BindingContext>
+    <local:ViewModel  x:Name ="viewModel"/>
+</ContentPage.BindingContext>
+
+<syncfusion:SfDataGrid x:Name="dataGrid"           
+                   ItemsSource="{Binding OrdersInfo}">
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridNumericColumn Format="C" HeaderText="ID"                                 MappingName="OrderID"/>
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+
+{% highlight c# tabtitle="MainPage.xaml.cs"%}
+dataGrid = new SfDataGrid();
+            
+DataGridNumericColumn numericColumn = new DataGridNumericColumn()
+{
+    MappingName = "OrderID",
+    HeaderText = "OrderID",
+};
+dataGrid.Columns.Add(numericColumn);
+{% endhighlight %}
+{% endtabs %}
+
+## Bind a view model property inside header template?
+
+The SfDataGrid allows binding the view model property to the `HeaderTemplate` by setting the BindingContext of the `DataGridColumn` as `ViewModel`.
+
+To bind a view model property inside `HeaderTemplate`, follow the code example:
```

Comment:
@NirmalKumarYuvaraj  - Remove this line.

Reply:
Removed the line.